### DISCUSSION TITLE
Nach firmware up/downgrade kein Zugriff auf dtu

### DISCUSSION THREAD

**Discussion by xxJAREKxx** [tbnobody/OpenDTU#1689] [Q&A]
Nach Update auf die aktuelle Version und anschließenden downgrade auf die Version vom 23 Dezember kann ich nicht mehr per WLAN auf die dtu zugreifen. Das Passwort ist angeblich falsch. 

Das ganze ist jetzt zum 2. Mal passiert auf 2 unterschiedlichen dtu.

Hat jemand eine Idee wie ich wieder Zugriff auf die dtu bekomme?

Vielen Dank

**Comment by Tiese** [comment]
Hallo @xxJAREKxx,
versuche es mal mit dem Standard-Passwort: `openDTU42`

**Comment by coltc50** [comment]
Ich habe das Problem nun auch... Habe nie einen Benutzer und Passwort erstellt.

Was ist den der Standardwert für Benutzer und Passwort?

**Reply by Tiese** [reply]
Hallo @coltc50,
der Benutzername lautet: `admin`

**Reply by coltc50** [reply]
Problem hat sich gelöst... Hatte doch ein eigenes Passwort vergeben... 😬
